extensions/skyportal/skyportal/app_server_fritz.py

```diff
@@ -25,6 +26,7 @@
     # Archive
     (r"/api/archive_catalogs", ArchiveCatalogsHandler),
     (r"/api/archive", ArchiveHandler),
+    (r"/api/cross_match", CrossMatchHandler),
```

Comment:
Thoughts on renaming the route to `/api/archive/cross_match`? Mostly since the `CrossMatchHandler` is under the `archive.py` umbrella. Seems to make a tiny bit more sense but I don't really have a preference one way or the other. Would make sense to go with `/api/archive/catalogs` as well in that case.

Reply:
Great idea, made the change.